apps/www/components/Sections/TwitterSocialProof.tsx

```diff
@@ -1,31 +1,35 @@
-import { useRef } from 'react'
-
+import { useState } from 'react'
 import { useRouter } from 'next/router'
-
-import { Swiper, SwiperSlide } from 'swiper/react'
-// import Swiper core and required modules
-import SwiperCore, { Navigation, Pagination } from 'swiper'
-
-import { Button, IconMessageCircle, IconArrowLeft, IconArrowRight } from 'ui'
-
-import Examples from '../../data/tweets/Tweets.json'
+import { Button, IconMessageCircle } from 'ui'
+import Tweets from '../../data/tweets/Tweets.json'
 import TweetCard from '../TweetCard'
-
-// Import Swiper styles
-import 'swiper/swiper.min.css'
-import 'swiper/components/navigation/navigation.min.css'
-import 'swiper/components/pagination/pagination.min.css'
 import Link from 'next/link'
 
-// install Swiper modules
-SwiperCore.use([Navigation, Pagination])
-
 function TwitterSocialProof() {
   // base path for images
   const { basePath } = useRouter()
 
-  const prevRef = useRef(null)
-  const nextRef = useRef(null)
+  const [tweets, setTweets] = useState(Tweets.slice(0, 10))
+  const [showButton, setShowButton] = useState(true)
+
+  const handleShowMore = () => {
+    setTweets((prevTweets) => [
+      ...prevTweets,
+      ...Tweets.slice(prevTweets.length, prevTweets.length + 10),
+    ])
+
+    if (tweets.length >= Tweets.length) {
+      setShowButton(false)
+    }
+
+    const container = document.querySelector('.tweet-container')
+    // @ts-ignore
+    container.style.height = `${container.clientHeight + 400}px`
```

Comment:
Not sure we need this. The container should adjust based on the content.

Reply:
Sadly the container doesn't adjust on its own. What happens when your remove the code above:

https://github.com/supabase/supabase/assets/70828596/327a8939-e2fe-491e-b3fb-04cf6daa7081

Reply:
@Isaiah-Hamilton took a shot at this, the problem was actually that the container had a fixed height. Now it actually grows with the content. 👍